rustler_tests/native/rustler_test/Cargo.toml

```diff
@@ -12,3 +12,5 @@ crate-type = ["cdylib"]
 [dependencies]
 lazy_static = "1.4"
 rustler = { path = "../../../rustler" }
+
+[workspace]
```

Comment:
Same comment as [here](https://github.com/rusterlium/rustler/pull/261/files#r336749182), this is probably not needed?